.eslintrc.js

```diff
@@ -2,5 +2,29 @@ module.exports = {
   globals: {
     __PATH_PREFIX__: true,
   },
-  extends: `airbnb`,
-}
+  parser: 'babel-eslint',
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: 'module',
+    ecmaFeatures: {
+      jsx: true,
+      modules: true,
+      experimentalObjectRestSpread: true,
+    },
+  },
+  extends: 'airbnb',
+  rules: {
+    'max-len': 'warn',
+    'no-unused-expressions': 'warn',
+    'react/jsx-filename-extension': [
+      'warn',
+      {
+        extensions: ['.js', '.jsx'],
+      },
+    ],
+    'jsx-a11y/click-events-have-key-events': 'off',
+    'jsx-a11y/control-has-associated-label': 'off',
+    'jsx-a11y/no-noninteractive-element-interactions': 'warn',
+    'jsx-a11y/no-static-element-interactions': 'warn',
+  },
```

Comment:
**Relaxed following linting rules:**
1. **max-len** _(maximum length of a line)_: Changed from **error** 🚩 to **warn** ⚠
2. [**no-unused-expressions**](https://eslint.org/docs/rules/no-unused-expressions) _(allows us to use short-circuiting, ternary operators in expressions & tagged-templates)_: Changed from **error** 🚩 to **warn** ⚠
3. **react/jsx-filename-extension** _(Allows us to use JSX markup inside a JS file rather than JSX file)_: Since Gatsby requires us to use JS files for pages, changed from **error** 🚩 to **warn** ⚠
4. [**jsx-a11y/click-events-have-key-events**](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/click-events-have-key-events.md): We need not enforce accompanying pointer-events with keyboard-events. Disabled 🚫
5. [**jsx-a11y/control-has-associated-label**](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/label-has-associated-control.md): We need not enforce control elements to be accompanied by labels. Disabled 🚫
6. [**jsx-a11y/no-noninteractive-element-interactions**](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-noninteractive-element-interactions.md): In some cases we need to assign interactions on non-interactive elements like the UL element when implementing pan. Changed from **error** 🚩 to **warn** ⚠
7. [**jsx-a11y/no-static-element-interactions**](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-static-element-interactions.md): We need not enforce static elems with interactions to have the **role** attribute defined coz it causes some other linting checks to fail. Changed from **error** 🚩 to **warn** ⚠

We could ofc revoke these relaxations once we have a concrete component structure definition in place 😊

Reply:
3. is fine off.
5. i would rather keep it as warn since is accessibility related and we want to be as accessible as possible.

Reply:
OK sounds good! Lemme update it 🙂